doubly_linked_list/doubly_linked_list.py

```diff
@@ -51,42 +54,102 @@ def __len__(self):
     """Wraps the given value in a ListNode and inserts it 
     as the new head of the list. Don't forget to handle 
     the old head node's previous pointer accordingly."""
+
     def add_to_head(self, value):
-        pass
+        new_node = ListNode(value, None, None)
```

Comment:
Good! Because the `ListNode` class has default args for `prev` and `next`, we don't have to include them here.